src/meta/src/barrier/creating_job/status.rs

```diff
@@ -89,7 +91,12 @@ impl CreatingStreamingJobStatus {
         &mut self,
         is_checkpoint: bool,
     ) -> Option<(
-        Vec<(TracedEpoch, TracedEpoch, BarrierKind)>,
+        Vec<(
```

Comment:
Shall we wrap it into a struct and also update the comments? This doesn't looks that readable.